design-at-cornell/src/homepage/Homepage.module.css

```diff
@@ -2,3 +2,22 @@
   height: 100%;
   width: 100%;
 }
+
+.Headline {
+  text-align: center;
+  font-family: 'Work Sans';
+  font-style: normal;
+  font-size: large;
+  text-align: center;
+  color: #0e1500;
+}
+
+.MissionStat {
+  text-align: center;
+  font-family: 'Work Sans';
+  font-style: normal;
+  font-size: medium;
+  text-align: center;
+  color: #0e1500;
+  padding-top: 20px;
+}
```

Comment:
We have been using StyledComponent as styling sheets. It works very similarly to css, you can look at how some of the other pages are implemented for reference. 

And we don't put the color hex code directly in the styling sheets because it's hard to change or to reuse the same color. We have a file `src/constants/colors.ts` that stores all color values. And then to access the colors in styling sheets, you can have `import { colors } from '../constants/colors';` and use `${colors.colorName}` in place of the hex code.